destination/destination.go

```diff
@@ -124,30 +142,30 @@ func (d *Destination) Teardown(ctx context.Context) error {
 // handleInsert checks for the existence of a key. If no key is present it will
 // plainly insert the data. If a key exists, but no key column name is
 // configured, it attempts a plain insert to that database.
-func (d *Destination) handleInsert(ctx context.Context, r sdk.Record) error {
+func (d *Destination) handleInsert(r sdk.Record, b *pgx.Batch) error {
```

Comment:
IIUC these methods don't actually execute the query anymore, they create the query and add it to a queue, so we should update the docs for each method accordingly